libautoscoper/src/Tracker.cpp

```diff
@@ -785,9 +789,9 @@ void Tracker::calculate_viewport(const CoordFrame& modelview,double* viewport) c
     for (int j = 0; j < 8; j++) {
 
         // Calculate the location of the corner in object space
-    corners[3 * j + 0] = (corners[3 * j + 0] - volumeDescription_[idx]->invTrans()[0]) / volumeDescription_[idx]->invScale()[0];
-    corners[3 * j + 1] = (corners[3 * j + 1] - volumeDescription_[idx]->invTrans()[1]) / volumeDescription_[idx]->invScale()[1];
-    corners[3 * j + 2] = (corners[3 * j + 2] - volumeDescription_[idx]->invTrans()[2]) / volumeDescription_[idx]->invScale()[2];
+        corners[3 * j + 0] = (corners[3 * j + 0] - volumeDescription_[idx]->invTrans()[0]) / volumeDescription_[idx]->invScale()[0];
```

Comment:
To ensure the git blame only reflects the introduced changes, let's avoid fixing the indent here.

We will use clang-format and perform a batch update along with updating [.git-blame-ignore-revs](https://github.com/BrownBiomechanics/Autoscoper/blob/main/.git-blame-ignore-revs)